projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -0,0 +1,125 @@
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import { __experimentalInspectorPopoverHeader as InspectorPopoverHeader } from '@wordpress/block-editor';
+import { Button, PanelRow, Dropdown, VisuallyHidden } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+import { useEntityProp } from '@wordpress/core-data';
+import { PluginPrePublishPanel } from '@wordpress/edit-post';
+import { PostVisibilityCheck } from '@wordpress/editor';
+import { __, sprintf } from '@wordpress/i18n';
+
+const visibilityOptions = {
+	public: {
+		label: __( 'Public', 'jetpack' ),
+		info: __( 'Visible to everyone', 'jetpack' ),
+	},
+	members_only: {
+		label: __( 'Subscribers', 'jetpack' ),
+		info: __( 'Visible to only subscribers', 'jetpack' ),
+	},
+	nobody: {
+		label: __( 'Private', 'jetpack' ),
+		info: __( 'Visible to only admins and editors', 'jetpack' ),
+	},
+};
+
+export default function NewsletterSettings() {
+	const [ postMeta, setPostMeta ] = useEntityProp( 'postType', 'post', 'meta' );
+	const visibilityKey = postMeta._newsletter_visibility ?? 'public';
+	const visibilityLabel = visibilityOptions[ visibilityKey ]?.label ?? 'Public';
+
+	return (
+		<>
+			<PanelRow>
+				<NewsletterAudience
+					setPostMeta={ setPostMeta }
+					visibilityKey={ visibilityKey }
+					visibilityLabel={ visibilityLabel }
+				/>
+			</PanelRow>
+
+			<PluginPrePublishPanel title={ __( 'Newsletter settings', 'jetpack' ) }>
```

Comment:
We should move this to this file: https://github.com/Automattic/jetpack/blob/1565ce1a20bf0eb2ef3a9b69181f11a21caa2175/projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js#L48

Reply:
Done